### ISSUE TITLE
Is it possible to define custom parameter for count and use it with loader?

### ISSUE THREAD

**Issue by fico7489** [nelmio/alice#1128]
For example if I have this fixtures:
```

Nelmio\Entity\User:
    user{1..10}:
        username: '<username()>'
```

I want to "10" be parameter with default 10, something like:

```
Nelmio\Entity\User:
    user{1..[CUSTOM_COUNT:10]}:
        username: '<username()>'
        fullname: '<firstName()> <lastName()>'
```

"CUSTOM_COUNT" is paramer with default 10.

When I call loader like this:

```
$loader = new Nelmio\Alice\Loader\NativeLoader();
$objectSet = $loader->loadFile(__DIR__.'/fixtures.yml');
```

I want to set that parameter

```
$loader = new Nelmio\Alice\Loader\NativeLoader();
$objectSet = $loader->loadFile(__DIR__.'/fixtures.yml', ['CUSTOM_COUNT' => 20]);
```

Is it possible something like that?

**Comment by alexislefebvre** [comment]
What if you load the same file twice?

```
$loader = new Nelmio\Alice\Loader\NativeLoader();
$objectSet = $loader->loadFile(__DIR__.'/fixtures.yml');
$objectSet = $loader->loadFile(__DIR__.'/fixtures.yml');
```

In theory, you should end up with 20 random users.

**Comment by fico7489** [comment]
Actually my default is 20 and in the test I need just few of them

**Comment by alexislefebvre** [comment]
A similar request #131 was fixed by #130, I don't know if it still works.

**Comment by theofidry** [comment]
Duplicate of #730; So it's not supported yet.
